roles/host-provision/tasks/proxy.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2020 Google LLC
+#
```

Comment:
It might also be worth noting (in documentation or comments) that this playbook is an implementation of https://cloud.google.com/vpc/docs/special-configurations#proxyvm

Reply:
Done